public/documents/fari-games/charge-srd.md

```diff
@@ -521,17 +598,18 @@ Once the mission is over, the PCs go back to their H.Q., and the loop starts ane
 > - _Finding dirt on the government to expose corruption._
 > - _Convincing the real king to take back what is theirs._
 > - _Finding money to hire mercenaries to fight an eventual war._
+>
+> Designing world forces is an efficient and simple way to fill the world with interesting factions that the players will be able to interact with.
 
-
-# Attachment | Extras
+# Attachment Extra | Extras
```

Comment:
Does it make sense to leave a designer note here about how attachments may change the momentum dial?